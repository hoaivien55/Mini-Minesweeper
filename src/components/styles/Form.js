import styled from 'styled-components';

const Form = styled.form`

    > * {
        margin-bottom: 15px;
    }

`

export default Form;