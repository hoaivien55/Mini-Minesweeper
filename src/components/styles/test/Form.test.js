import renderer from "react-test-renderer";
import Form from "../Form";

it("Form: Virtual DOM snapshot", () => {
	const tree = renderer.create(<Form />).toJSON();
	expect(tree).toMatchSnapshot();
});
