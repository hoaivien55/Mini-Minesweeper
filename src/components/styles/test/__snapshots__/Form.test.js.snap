// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form: Virtual DOM snapshot 1`] = `
.c0 > * {
  margin-bottom: 15px;
}

<form
  className="c0"
/>
`;
